### ISSUE TITLE
Unable to paste an image to chat from the clipboard

### ISSUE THREAD

**Issue by Grief** [raelgc/scudcloud#563]
## ScudCloud Version

ScudCloud 1.52
Python 3.5.3
Qt 5.7.1
PyQt 5.7
SIP 4.18.1

## Distro and Desktop info
**Ubuntu Zesty+KDE Neon**

## ISSUE ##
I cannot CTRL+V an image from the clipboard to the chat anymore since one of the recent Qt updates (probably). Also, I cannot get context menu with right click in the message text box (that which contains 'cut', 'copy' and 'paste'). Nothing happens when I click text box with right button, until I select some text.

**Comment by raelgc** [comment]
You're right, more one side effect from the migration to Qt5 :(

I'll investigate.

**Comment by Grief** [comment]
@raelgc I was one of the first who tried qt5 port and it was working well. I suspect that something changed in Qt5 recently. By the way, I remember there was an issue with spell checking in qt5 - mistakes were not highlighted with red. I see highlighting now but it is totally misplaced.

**Comment by Grief** [comment]
@raelgc your fix for right click is working now :+1: , but I am still not able to paste images :(

**Comment by raelgc** [comment]
@Grief Yeap, Qt version included in Ubuntu received several "fixes", and they broken a lot of working stuff. Mainly the image pasting, red marking and zoom scale.

**Comment by raelgc** [comment]
Interestingly, if you paste the image while the cursor is not in the message area, it'll work.

**Comment by Grief** [comment]
@raelgc I confirm that. For me this workaround is sufficient, thanks!

**Comment by cayetanobv** [comment]
@raelgc thanks for this workaround! (https://github.com/raelgc/scudcloud/issues/563#issuecomment-293376019)

**Comment by raelgc** [comment]
Soon I'll publish a fix for it (it's ready, waiting to be merged and published).

**Comment by cayetanobv** [comment]
@raelgc these is awesome!! Thanks again for your great work with scudcloud.

**Comment by cayetanobv** [comment]
@raelgc @Grief With 1.57 it works fine again (with the cursor in the message area). Thanks.

**Comment by raelgc** [comment]
Thank you all for raising this issue and the feedback!

**Comment by akanarsky** [comment]
@raelgc it is not working again in 1.65 (I actually tried master) - Linux Mint 18.3 
The workaround works though (paste when the cursor not in the message area)

**Comment by raelgc** [comment]
@akanarsky Slack is constantly changing their JS code, which is hard to follow. Need to debug and investigate the reason why it's not accepting the content when pasting with cursor in the message area.

Can you open a separate issue, detailing it's happening only when cursor is in the message input?
